### ISSUE TITLE
Tests running twice on each PR

### ISSUE THREAD

**Issue by pvandyken** [akhanf/snakebids#167]
Noticed this has been happening lately. I wonder if it's because both PRs and Pushes are listed as instigating events, and maybe only Pushes are needed? @kaitj

**Comment by kaitj** [comment]
Is this happening when the PR is merged? Whenever it is happening, it is likely because both PRs and Pushes are listed as you said. I think we can be more explicit in the actions workflow about the triggers for each by using `types`. I think what may be happening right now is both the `push` event, as well as the `synchronize` event for PRs are getting triggered.

**Comment by pvandyken** [comment]
I'm not totally sure what the pattern is, tbh. It's not just when it's merged, though. Check the current open PR for an example. It's not a massive issue, just something that came up recently. We can observer longer for a pattern if necessary

**Comment by pvandyken** [comment]
Just to close this off properly, I now understand why this issue occurs. It happens, e.g. when making a change directly on Github and making a PR. Github creates a new branch and pushes your changes to that branch, triggering one instance of tests. Then, it creates a PR from your branch, triggering the second instance of tests. Because both of these things happen "simultaneously", they all get grouped and summarized under the PR results panel, but in reality, both sets of tests always occur. The "push" tests are just usually in the background.

Long story short: nothing to be concerned about here
